### ISSUE TITLE
Show the blacklist after installation (avoid the risk of loss)

### ISSUE THREAD

**Issue by grahamperrin** [AXeL-dev/distract-me-not#85]
Feature request: 

* after installation, please show the blacklist.

----

Without this feature: data loss may occur, as outlined below.

### Your environment

- extension version 2.8.0
- Firefox 108.0.1

```text
% pkg info -x firefox
firefox-108.0.1_2,2
% uname -aKU
FreeBSD mowa219-gjp4-8570p-freebsd 14.0-CURRENT FreeBSD 14.0-CURRENT #28 main-n259865-b079ca8593ed-dirty: Tue Dec 27 15:57:33 GMT 2022     grahamperrin@mowa219-gjp4-8570p-freebsd:/usr/obj/usr/src/amd64.amd64/sys/GENERIC-NODEBUG amd64 1400077 1400077
% 
```

### Steps to reproduce

1. before discovering the extension
2. set Firefox to open previous windows and tabs
3. have many YouTube tabs in the session (maybe non-loaded, maybe not visible)
4. encounter a website (not YouTube) that should be blocked
5. discover and install the extension
6. use the context menu to block the site
7. enable the extension, to verify the block

### Expected behaviour

8. the required block
9. quit
10. start Firefox
11. observe previous windows and tabs

### Actual behaviour

8. the required block – plus **unwanted blocks of required tabs** (maybe not apparent)
9. quit
10. start Firefox
11. observe a subset of previous windows and tabs – **all YouTube tabs are lost**.

**Comment by grahamperrin** [comment]
### UX: a real-world encounter

I was lucky. 

With a multi-window session, after enabling the extension in a window that had no YouTube tabs, I switched to a different window and then observed unexpected appearances of the Distract Me Not icon: 

![2023-01-02 17-11-09](https://user-images.githubusercontent.com/192271/210291278-d7d2e3ca-aa0a-4437-a71a-83c0e2966e9a.png)

After quitting then starting the browser: 

![2023-01-02 17-22-10](https://user-images.githubusercontent.com/192271/210291349-7e04f8b8-feb7-435f-bb36-be99a6f1eee1.png)

* adjacent tabs were present
* tabs that were caught by the extension were lost.

I was lucky, in that: 

* the unwanted catches were observed in good time
* I had an automated [Simple Tab Groups](https://addons.mozilla.org/addon/simple-tab-groups/) backup from not long before the loss. 

After restoration (a filtered view of `youtube.com` tabs with [Tab Manager Plus](https://addons.mozilla.org/addon/tab-manager-plus-for-firefox/)): 

![2023-01-02 17-41-28 restored](https://user-images.githubusercontent.com/192271/210291557-811154fc-ea88-497e-9d40-e40f5028a0b4.png)

**Comment by AXeL-dev** [comment]
Hey @grahamperrin & thank you for reporting the issue,

I do totally agree that this is a UX flaw, actually the blacklist should be empty by default, i don't remember why i filled it with domains like `youtube.com` but it definitely should be filled by the user & not come filled by default.

Kudos for the wonderful idea of opening the blacklist after installation, i love it!

**Comment by AXeL-dev** [comment]
**Remaining**: open the settings/blacklist page after installation.

**Comment by AXeL-dev** [comment]
Fixed on release `2.8.1`, the blacklist is now empty so there is no need to automatically open it after installation.
